### PR TITLE
HIP-584: DatabaseAccessor implementations for StackedStateFrame

### PR DESCRIPTION
**Pull request by mustafauzunn**
**Description**:

DatabaseAccessor implementations for StackedStateFrame. This is where you go to the database for requested entities

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/CustomFeeDatabaseAccessor.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+@Named
+@RequiredArgsConstructor
+public class CustomFeeDatabaseAccessor extends DatabaseAccessor<Long, List<CustomFee>> {
+
+    private final CustomFeeRepository customFeeRepository;
+
+    @Override
+    public @NonNull Optional<List<CustomFee>> get(@NonNull Long key) {
+        return Optional.ofNullable(customFeeRepository.findByTokenId(key));
```
Comment (steven-sheehy) [inline hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/CustomFeeDatabaseAccessor.java:35]: Technically a repository can never return a null list.

**Review by steven-sheehy — APPROVED** [review]
LGTM

File: hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/CustomFeeDatabaseAccessor.java (hunk 1 above)
Comment (david-bakin-sl) [reply hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/CustomFeeDatabaseAccessor.java:35]: Maybe `CustomFeeRepositoryExtra.findByTokenId` should be so annotated?

**Review by david-bakin-sl — APPROVED** [review]
LGTM.  Makes these accessors look easy.

**Pull request merged**